### ISSUE TITLE
Help wanted : TensorboardX

### ISSUE THREAD

**Issue by christiantheriault** [zylo117/Yet-Another-EfficientDet-Pytorch#614]
First, thank you for this GREAT work. I've never used TensorboardX. Can we visualize loss and regression curves with it ? If so, can you point me in the good direction to use it ? Thanks !

**Comment by zylo117** [comment]
1. run command `tensorboard --logdir /path/to/logs/project_name/tensorboard/`
2. open link `http://127.0.0.1:6006` with browser

**Comment by christiantheriault** [comment]
Thanks ! That worked well. I augmented the COCO dataset (images and json files) with 2 other person-only datasets (Humancrowd and Widerperson). I filtered the original json file to keep only the person category. Do you think it will make a difference in training if I keep 80 units/categories at the top, or should I reduce it to 1 unit ? If I keep 80 units, the other 79 units will simply not be trained/used ?

If I change 80 to 1 in all the init files and chose only person in the config, when training head only, I get this warning. And the initial classification error is much larger than if I keep the parameter at 80.

[Warning] Ignoring Error(s) in loading state_dict for EfficientDetBackbone:
        size mismatch for classifier.header.pointwise_conv.conv.weight: copying a param with shape torch.Size([810, 112, 1, 1]) from checkpoint, the shape in current model is torch.Size([9, 112, 1, 1]).
        size mismatch for classifier.header.pointwise_conv.conv.bias: copying a param with shape torch.Size([810]) from checkpoint, the shape in current model is torch.Size([9]).
[Warning] Don't panic if you see this, this might be because you load a pretrained weights with different number of classes. The rest of the weights should be loaded already.

**Comment by zylo117** [comment]
1. ignore the warning if you know what you are doing
2. less classes is always better

**Comment by christiantheriault** [comment]
Thanks ! Sorry for the late reply ! Can we donate through PayPal ?

**Comment by christiantheriault** [comment]
Hello :)

Regarding post-processing (utils/utils.py) in EfficientDet with batch size 8 for example.

In eval mode with batch size 8, the first iteration takes over 30 milliseconds  whereas each of the next 7 iteration takes around 1 millisecond.  Would you know how to get around this ?
33.74791145324707
0.8144378662109375
0.6785392761230469
0.64849853515625
0.6930828094482422
1.2445449829101562
3.2536983489990234
0.591278076171875
